src/modules/services/rootEndpoints.js

```diff
@@ -1,4 +1,5 @@
 export const SHOWS_API = 'https://api.tvmaze.com/shows';
 export const INVOLVEMENT_API = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
-export const LIKES = 'IONInnzNi3VRPNlFloAi/likes'
-export const COMMENTS = 'IONInnzNi3VRPNlFloAi/comments'
\ No newline at end of file
+export const APP_ID = 'UA1qh5oDK24p7rhA1W5m'
```

Comment:
[OPTIONAL]

- I will strongly recommend you always store your sensitive data such as the` APP_ID` in an environment variable to prevent anyone from using it for any malicious activity that might impact your website 😄